lib/sidekiq_flow/client.rb

```diff
@@ -1,39 +1,42 @@
 module SidekiqFlow
   class Client
-
     class << self
+      attr_accessor :adapters
+      attr_reader :configuration
```

Comment:
we already have the method configuration, no need for a reader.

attr_reader is just a shortcut for
```
  def configuration
     @configuration
  end
```